### DISCUSSION TITLE
@headlessui/react@next set maxHeight on ListboxOptions

### DISCUSSION THREAD

**Discussion by hauptrolle** [tailwindlabs/headlessui#3025] [Help]
Hey, 

does anyone has managed to set a `maxHeight` to the new ListboxOptions component? (`@headlessui/react@next`).
It get's an inline style of `maxHeight` which always fills the whole available space. That is quite annoying :( 

Any help is appreciated ✌️

**Comment by bastianccm** [comment]
Seems to be caused by the floating-ui sizeMiddleware:
https://github.com/tailwindlabs/headlessui/blob/043edb83196dde5c593ff9d615b5d89e35f8b87d/packages/%40headlessui-react/src/internal/floating.tsx#L295-L301

I'm not familiar with floating-ui, but maybe exposing the floating Context might help?

**Comment by hauptrolle** [comment]
@RobinMalfait Hey 👋 Can we get some feedback on this one?

**Comment by ariaspabloi** [comment]
You can use the the !important css rule, is not recommend to use but I could not find any other fix.
Using tailwind in react:

```
 <ListboxOptions className="!max-h-80">
```
